addon/views/modal.js

```diff
@@ -33,6 +34,23 @@ export default Em.View.extend(
     this.set('visible', true);
   },
 
+  /* Hide with Escape */
+  turnOnEscapeEvent: function() {
+    var _this = this;
+
+    this.set('keydownListener', function(event) {
```

Comment:
Why is this set as a property on the view here? It could just be a variable in this function or a property from the get go. For example:

```
export default Ember.View,extend({
  // ...

  onEsc: function(event) {
    if (event.which === _this.escapeKeyCode) {
      // ...
    }
  },

  turnOnEscapeEvent: function() {
    Em.$('body').on('keydown', this.get('onEsc'));
  }
});
```